internal/objects/ingress_to_service.go

```diff
@@ -361,20 +361,27 @@ func (v *SvcNSCache) UpdateIngressMappings(ingName string, svcName string) {
 	}
 }
 
-func (v *SvcNSCache) UpdateIngressSecretsMappings(ingName string, secret string) {
+func (v *SvcNSCache) AddIngressToSecretsMappings(ingressNS, ingName, secretName string) {
```

Comment:
Since this function adds secret to mapping, probably we should change the name to: `AddSecretToIngressMappings`.